### PR TITLE
Fix int8x4 broadcast value codegen in cuda

### PR DESCRIPTION
**Pull request by vinx13**
There are no `make_int` or `make_int8x4` in cuda, so we need to generate packed value in codegen. This PR fixed codegen of `x4(int8)imm_value` in cuda.

### CODE REVIEW

**Comment by vinx13** [comment]
@tqchen Please review.

**Comment by tqchen** [comment]
@nishi-t can you help review this?

File: tests/python/unittest/test_codegen_cuda.py [hunk 1]
```diff
@@ -87,7 +87,29 @@ def check_cuda(dtype, n, lanes):
...
+        y, x = s[A].op.axis
+        s[A].vectorize(x)
+        s[A].bind(y, tvm.thread_axis("blockIdx.x"))
+        fun = tvm.build(s, [A], "cuda", name="make_int8x4")
+        np_a = np.full((n, lanes), value, dtype=dtype)
+        a = tvm.nd.empty(np_a.shape, dtype, ctx)
+        fun(a)
+        np.testing.assert_equal(a.asnumpy(), np_a)
+    check_cuda(64, 0xAB)
+    check_cuda(64, 0)
```
Comment (nishi-t) [inline tests/python/unittest/test_codegen_cuda.py]: @vinx13 Thanks. Could you add a test case for negative value?

**Comment by tqchen** [comment]
Thanks @nishi-t for reviewing, Thanks @vinx13 for contribution! This is merged
